packages/plugins/GoPlusSecurity/src/SNSAdaptor/CheckSecurityDialog.tsx

```diff
@@ -42,41 +60,65 @@ export function CheckSecurityDialog(props: BuyTokenDialogProps) {
     const t = useI18N()
     const classes = useStylesExtends(useStyles(), props)
     const { open, onClose } = props
+    const [chainId, setChainId] = useState<ChainId>()
+
+    const [{ value, loading: searching, error }, onSearch] = useAsyncFn(
+        async (chainId: ChainId, content: string): Promise<TokenSecurity | undefined> => {
+            if (!content || isSameAddress(content.trim(), ZERO_ADDRESS)) return
+            setChainId(chainId)
+            let values = await GoPlusLabs.getTokenSecurity(chainId, [content.trim()])
+            values ??= {}
+            if (isEmpty(values)) throw new Error('Contract Not Found')
```

Comment:
i18n too.

Reply:
why error message needs i18n

Reply:
> why error message needs i18n

If a message would be displayed on the screen, then it should be represented in i18n.